e2e/suites/management/create_new_scheduler_version_test.go

```diff
@@ -45,11 +45,13 @@ import (
 func TestCreateNewSchedulerVersion(t *testing.T) {
 	game := "create-new-scheduler-version-game"
 	framework.WithClients(t, func(roomsApiClient *framework.APIClient, managementApiClient *framework.APIClient, kubeClient kubernetes.Interface, redisClient *redis.Client, maestro *maestro.MaestroInstance) {
-		t.Run("Should Succeed - create minor version, no pods replaces", func(t *testing.T) {
+		t.Run("Should Succeed - create minor version, no pods replaces, create major version, all pods are changed", func(t *testing.T) {
```

Comment:
Insane gameplay mechanics on e2e holy christ

Reply:
For testing the new scenarios i had to basically create a minor and a major change to setup environment, so I thought it wouldn't make sense to have 2 tests basically testing the same thing